scripts/pum

```diff
@@ -527,7 +526,8 @@ if __name__ == "__main__":
     parser_test_and_upgrade.add_argument(
         '-u', '--max-version', help='upper bound limit version')
     parser_test_and_upgrade.add_argument(
-        '-v', '--var', nargs=2, help='Assign variable for running SQL deltas.',
+        '-v', '--var', nargs=3, help='Assign variable for running SQL deltas.'
+                                     'Format is: (string|float|int) name value.',
         action='append', required=False)
```

Comment:
To restate what I said in https://github.com/opengisch/pum/pull/41: `required=False` is not necessary, and you could use `default=[]`.